scripts/baseline_comparison/evaluate_baselines.py

```diff
@@ -209,6 +210,28 @@ def run_evaluate_baselines(
     else:
         print(f"Skipping expensive simulations because `run_expensive_simulations={run_expensive_simulations}`")
 
+    if run_max_models_simulations:
```

Comment:
Not necessarily for this PR but we should tag in our README.md the version recommended to reproduce our results since we are modifying the main scripts and will likely introduce some change compared to the results in the paper.

Reply:
Added to README and created `AutoML2024` branch for this